benchmark/single-source/SetTests.swift

```diff
@@ -68,32 +68,32 @@ public let SetTests = [
     tags: [.validation, .api, .Set],
     setUpFunction: { blackHole([setAB, setE]) }),
   BenchmarkInfo(
-    name: "SetIsSubsetInt0",
```

Comment:
Unfortunately we shouldn't change the names of existing benchmarks here -- we're using these to track performance across compiler releases. 

It is a good idea to proper names for the new benchmarks, though.

Reply:
Thank you for your advice.
I fixed it.